js/crm.menubar.js

```diff
@@ -336,7 +336,33 @@
               CRM.menubar.open('QuickSearch');
             }
           }
-        });
+        })
+        .autocomplete( "instance" )._renderItem = function( ul, item ) {
+          var content;
+          if (item.value == 0) {
+            content = item.label;
```

Comment:
I believe this function outputs raw html, so this would need to be:

```suggestion
            content = _.escape(item.label);
```

Reply:
I tested this PR with an ampersand and angle brackets and it displayed ok, so maybe it's getting escaped somewhere else?

Reply:
Oh wait this is only if value = 0, and I don't know what that means.

Reply:
Anyway otherwise the PR works for me.